temporal-sdk/src/main/java/io/temporal/workflow/Workflow.java

```diff
@@ -1181,6 +1183,19 @@ public static Logger getLogger(String name) {
     return WorkflowInternal.getLastCompletionResult(resultClass, resultClass);
   }
 
+  /**
+   * Extract the latest failure from some previous run for this cron workflow. This is used in
```

Comment:
This doesn't apply to cron workflows only. It applies to any workflow retries (at least this is my understanding that is worth testing). So I would say something like:

Extract the latest failure from some previous run for this workflow. This run can be created as a retry of the previous failed run or as a next cron invocation for cron workflows.